install/install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+set -e
+
+export SPACK_VERSION=develop
+
+# Install Spack via repository clone
+git clone -c feature.manyFiles=true -c core.sharedRepository=true https://github.com/spack/spack.git
+pushd spack
+git checkout ${SPACK_VERSION}
+popd
+
+# Initialize spack
+. spack/share/spack/setup-env.sh
+
+# Create and activate chiltepin environment
+spack env create chiltepin
+spack env activate chiltepin
+
+# Set up read-only mirror
+spack mirror add --s3-access-key-id "" --s3-access-key-secret "" s3_spack_stack_buildcache_ro s3://chiltepin/spack-stack/
+if [ -n "${S3_BUILD_CACHE_KEY}" ]; then
+  cat mirrors.in.yaml | envsubst > spack/etc/spack/mirrors.yaml
+  chmod 600 spack/etc/spack/mirrors.yaml
+fi
+spack mirror list
+
+# Add flux and pytest to chiltepin environment
+spack add python
+spack add py-pip
+spack add py-pytest
+spack add flux-core@0.53.0
+spack add flux-sched@0.28.0
+
+# Concretize and install
```

Comment:
`install ___`? The spack?

Reply:
Thanks for the feedback.  I've pushed some updates.  I'll need you to re-approve if you like the changes.